### DISCUSSION TITLE
Welcome to StockAlertBot Discussions!

### DISCUSSION THREAD

**Discussion by Prince25** [Prince25/StockAlertBot#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by handsomegaz** [comment]
Just donated and wanted to thank you! This works perfectly and should save me a lot of time! :)

**Reply by Prince25** [reply]
Thank you so much. I'm glad it's working well for you and it's serving your needs like it did for me.

**Comment by elypsis** [comment]
Great piece of work, running on an AWS instant. Would it be possible to add ebuyer in UK? (ebuyer.com) 

Many thanks in advance

**Reply by Prince25** [reply]
Thank you. Please create a new discussion for requests and I'll look into it for next update!

**Reply by Prince25** [reply]
Hello. Just letting you know that @Jeisonx has added Ebuyer for you 😃
